component/namespace-policies.jsonnet

```diff
@@ -58,6 +58,36 @@ local organizationNamespaces = kyverno.ClusterPolicy('organization-namespaces')
     validationFailureAction: 'enforce',
     background: false,
     rules: [
+      {
+        name: 'set-default-organization',
+        match: {
+          resources: {
+            kinds: [
+              'Namespace',
+            ],
+          },
+        },
+        context: [
+          {
+            name: 'ocpuser',
+            apiCall: {
+              urlPath: '/apis/user.openshift.io/v1/users',
+              jmesPath: "items[?metadata.name=='{{ request.userInfo.username}}']",
+            },
+          },
+        ],
+        exclude: common.BypassNamespaceRestrictionsSubjects(),
+        mutate: {
+          patchStrategicMerge: {
+            metadata: {
+              labels: {
+                '+(appuio.io/organization)':
+                  '{{ocpuser[0].metadata.annotations."appuio.io/default-organization"}}',
```

Comment:
Should we make the annotation key (`appuio.io/default-organization`) configurable in the component?

Reply:
Wouldn't hurt I guess, but I don't see an immediate need either.